rusoto/core/src/lib.rs

```diff
@@ -35,6 +35,7 @@ pub extern crate rusoto_credential as credential;
 extern crate serde;
 #[macro_use]
 extern crate serde_derive;
+pub extern crate flate2;
```

Comment:
What do you think about it? People will access it as `rusoto_core::flate2`

Reply:
I don't thin we need to make it `pub`. If people need it they will defined their own dependencies.